sdk/core/core-http/src/serializer.ts

```diff
@@ -791,7 +793,7 @@ function deserializeDictionaryType(
   responseBody: any,
   objectName: string
 ): any {
-  /*jshint validthis: true */
+  /* jshint validthis: true */
```

Comment:
does this comment even do anything?

Reply:
I didn't add this. I think prettier added the space.

Reply:
Yeah, but maybe we can clean these comments up since we're not using jshint?

Reply:
ohh I get your point